src/kernel/kernel.ts

```diff
@@ -2499,12 +2499,12 @@ export function Boot(fsType: string, fsArgs: any[], cb: BootCallback, args: Boot
 					cb(err, undefined);
 					return;
 				}
-				let writable = new bfs.FileSystem['InMemory']();
+				let writable = new bfs.FileSystem['LocalStorage']();
```

Comment:
will you please add a boolean flag to `BootArgs` to choose between `LocalStorage` and `InMemory`?  There are different applications that use the Browsix Kernel, including the latex example as well as the shell.  We still want the shell to use `InMemory`, and only switch to `LocalStorage` when `args.UseLocalStorage` is true.